content/en/blog/falco-modern-bpf/index.md

```diff
@@ -0,0 +1,206 @@
+
+If you have a project that needs BPF instrumentation for one of these architectures you could simply link the Falco libraries (`libsinsp`, `libscap`) to obtain a working solution out of the box.
+
+> At this moment, we ship Falco for `x86_64` and `arm64` architectures only, due to their popularity in the community.
+
+## Requirements ⛓️
+
+To use the modern BPF probe, there are 2 main requirements:
+
+1. **A kernel that implements all the aforementioned features**. Linux kernel version `5.8` is the first kernel that supports **all of them**, and for this reason, we consider it the minimum required one.
+
+    Nevertheless, these features could also be backported into older kernels, so it wouldn't be completely fair to define the `5.8` as the first supported version. This is just a strict assumption that we put in place since we still miss the logic to detect all the necessary features of your kernel.
+
+    Until that day arrives, if your kernel is older than `5.8`, you might find the following error when trying to start the modern probe:
+
+    ```txt
+    Error: Actual kernel version is: '5.4.0' while the minimum required is: '5.8.0'
+    ```
+
+2. **A kernel that exposes BTF types**. This shouldn't be a big issue since we already require a kernel version newer than `5.8` and most [recent Linux distributions](https://github.com/libbpf/libbpf#bpf-co-re-compile-once--run-everywhere) come with kernel BTF capabilities.
+
+    If you want to be sure you can easily check their presence by typing:
+
+    ```bash
+    ls /sys/kernel/btf/vmlinux
+    ```
+
+    If your kernel supports them you should see:
+
+    ```bash
+    /sys/kernel/btf/vmlinux
+    ```
+
+Now, if your machine satisfies these 2 requirements, you are ready to have fun with the modern probe! 🚀
+
+## Modern BPF in action 🏎️
+
+Falco provides you with pre-release packages to try the modern BPF probe, but the full release will take place in Falco 0.34.
+
+If you happen find some bugs or misbehaviors, please be kind with us and [open an issue on Falco](https://github.com/falcosecurity/falco/issues) 🙏
+
+Having said that, you have 3 main ways to try Falco with the modern probe.
+
+### 1. Pre-built `deb, rpm` packages
+
+You can find [here](https://app.circleci.com/pipelines/github/falcosecurity/falco/3453/workflows/c8573555-0ecb-44de-af84-1f2a4121d772/jobs/29647/artifacts) the `x86_64` packages and [here](https://app.circleci.com/pipelines/github/falcosecurity/falco/3453/workflows/c8573555-0ecb-44de-af84-1f2a4121d772/jobs/29646/artifacts) the `arm64` ones.
+
+Taking as an example the `deb` package, you have simply to download it and type the following command:
+
+```bash
+sudo dpkg -i <your_deb_package.deb> 
+```
+
+Now there are 2 possibilities:
+
+1. If you have the `dialog` binary installed on your system you should see a dialog window like this:
```

Comment:
Perhaps a NOTE about the fact that this dialog is based upon https://github.com/falcosecurity/falco/pull/2242, so that users won't ask "why i haven't seen it before?" or think it comes only with the new probe :)